test/e2e/common/node/expansion.go

```diff
@@ -106,7 +106,7 @@ var _ = SIGDescribe("Variable Expansion", func() {
 		Testname: VolumeSubpathEnvExpansion, subpath expansion
 		Description: Make sure a container's subpath can be set using an expansion of environment variables.
 	*/
-	framework.ConformanceIt("should allow substituting values in a volume subpath [sig-storage]", func() {
+	framework.ConformanceIt("should allow substituting values in a volume subpath", func() {
```

Comment:
We're trying to ensure that each test has exactly one owning SIG.

If any of the tests shouldn't be owned by sig-node (and instead owned by sig-storage) - please let me know.

Reply:
FYI @msau42 @saad-ali